config/settings.yml

```diff
@@ -835,6 +835,13 @@ genisis:
   user: TestUser
   pass: bogus
 
+# Settings for the VEText integration (mobile push notifications)
+vetext_push:
+  va_mobile_app_sid: EC15A7E1767C4018AC50B442AAC50D1E # This is a placeholder. Should be replaced with real SID
+  base_url: https://vetext1.r01.med.va.gov
+  user: vets-api-username
+  pass: secret
```

Comment:
These all all placeholders, is there value in setting these to "real-looking" data?